js/core/src/init.ts

```diff
@@ -399,7 +400,7 @@ export function create_components(
 			);
 
 			instance.props.initial_tabs = child_tab_items?.map((child) => ({
-				label: child.props.label,
+				label: child.props.label.includes(i18n_marker) ? "" : child.props.label,
```

Comment:
Made this change so that the entire i18n dict does not show up but tbh it is on the page so briefly I think we can also not do this